concepticondata/concepticon.tsv

```diff
@@ -3596,3 +3596,97 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3610	MALLARD	Animals	A member of the Anatidae family, common in large parts of the world and native to the northern hemisphere.	Person/Thing	
 3611	AMERICAN BLACK BEAR	Animals	A member of the Urisdae family (Ursus americanus) native to North America.	Person/Thing	
 3612	INDIGENOUS PEOPLES OF THE AMERICAS	Social and political relations	Referring to the native inhabitants of North and South America.	Person/Thing	
+3613	LOWER BACK	The body	The lower part of the spine, also known as the lumbar region, which is situated between the rib cage and the pelvis.	Person/Thing	
+3614	BONE MARROW	The body	The spongy tissue in the hollow parts of bones.	Person/Thing	
+3615	PANT	The body	To breathe quickly and loudly as a result of running or exercising.	Action/Process	
+3616	BLEED	The body	To lose blood because of a disease or an injury.	Action/Process	
+3617	FAINT (PASS OUT)	The body	To become unconscious as a result of a temporary decrease in the blood supply to the brain.	Action/Process	
+3618	STUMBLE	Motion	To step awkwardly or to catch your foot on something and almost fall.	Action/Process	
+3619	STAMP	Motion	To put your foot down on the ground heavily and noisily; to walk in this way.	Action/Process	
+3620	LEAN	Basic actions and technology	To bend or move from a vertical position or to make something do that.	Action/Process	
+3621	SLEEPY	The body	Feeling tired and willing to go to sleep.	Property	
+3622	DISABLED PERSON	The body	A person who has an illness or an injury that restricts the way they live their life, especially by making it difficult for them to move.	Person/Thing	
+3623	BURN (INJURY)	The body	An injury, caused by exposure to heat, fire or chemical agents.	Person/Thing	
+3624	INTESTINAL WORM	Animals	The organism, also known as the parasitic worm, that infects and breeds in the gastrointestinal tract of humans and animals.	Person/Thing	
+3625	LEPROSY	The body	The infectious disease, caused by the bacteria Mycobacterium leprae, that affects the skin, as well as the peripheral nerves.	Person/Thing	
+3626	HEADACHE	The body	A pain in the head.	Person/Thing	
+3627	KNOW	Cognition	To have met and recognize somebody or to be certain or sure about something.	Action/Process	
```

Comment:
Hm, we have `1410 KNOW (SOMETHING) To be certain or sure about something.` Would this be more general? What is the difference?

Reply:
kennen + wissen, know facts and know a person -> know facts or a person

Reply:
Yes, the idea behind the new concept is to create a hypernym of KNOW (SOMETHING) and KNOW (SOMEBODY) because some languages don't have separate lexemes for the two (e.g., in Russian) and you can find something like _know something or somebody_ as an entry.